src/pycall/julia_helpers.jl

```diff
@@ -0,0 +1,25 @@
+using PyCall
+using AwkwardArray
+
+const ak = pyimport("awkward")
+const np = pyimport("numpy")
+
+function julia_array_to_numpy(array)
```

Comment:
does this need to be type lock down so it can ony take primitive awkward arrays?